superset-frontend/src/explore/components/DataTablesPane/components/ResultsPaneOnDashboard.tsx

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 import React from 'react';
-import { t } from '@superset-ui/core';
+import { t, css } from '@superset-ui/core';
```

Comment:
@codyml Thanks for the fixing, there is a simple pattern for wrapping the existing components.

```typescript
const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  height: 100%;
  .ant-tabs {
    height: 100%;
  }
  .ant-tabs-content {
    height: 100%;
  }
  .ant-tabs-tabpane {
    display: flex;
    flex-direction: column;
  }
  .table-condensed {
    overflow: auto;
  }
`;
....
....
    return <Wrapper>{resultsPanes[0]}</Wrapper>;
....
....
    <Wrapper><Tabs fullWidth={false}> {panes} </Tabs></Wrapper>



```

Reply:
Will do, thanks!